internal/kafka/internal/services/clusters_test.go

```diff
@@ -241,6 +251,35 @@ func Test_GetClusterDNS(t *testing.T) {
 			},
 			wantErr: true,
 		},
+		{
+			name: "failed to get cluster DNS from OCM",
```

Comment:
Same comment as https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pull/981/files#r862961723 adapted to this scenario